test/CheckoutTest.py

```diff
@@ -115,7 +109,8 @@ def test_payments_error_mocked(self):
                     'expiryMonth': '08',
                     'type': 'scheme',
                     'cvc': '737'
-                }
+                },
+                'applicationInfo': {'adyenLibrary': {'name': 'adyen-python-api-library', 'version': '7.1.1'}}
```

Comment:
```suggestion
                'applicationInfo': {'adyenLibrary': {'name': 'adyen-python-api-library', 'version': settings.LIB_VERSION}}
```
Hardcoding will break the tests when upgrading version